packages/relayer/processor/processor.go

```diff
@@ -205,7 +205,7 @@ func InitFromConfig(ctx context.Context, p *Processor, cfg *Config) error {
 			return err
 		}
 
-		// only support one hop rn, add in array configs
+		// only support one hop right now, add in array configs
```

Comment:
Could you undo this modification and change the title of the PR to something like "update solidity doc link to *latest*.

Reply:
done